File: week3/Student.cpp
#include<iostream>
using namespace std;

class Student
{
    public:
        string fullName;
        int rollNum;
        double semPerentage;
        string collegeName;
        int collegeCode;
        Student()
        {
            cout<<"Student class initialized!!\nIam a student\n";
        }
        ~Student(){}
};

int main()
{
    Student venkat;
    cout<<"Enter your name : ";
    getline(cin,venkat.fullName);
    cout<<"Enter your roll number : ";
    cin>>venkat.rollNum;
    cout<<"Enter your sem percentage : ";
    cin>>venkat.semPerentage;
    cout<<"Enter your college name : ";
    cin>>venkat.collegeName;
    cout<<"Enter your college code : ";
    cin>>venkat.collegeCode;
    cout<<"Name : "<<venkat.fullName<<endl;
    cout<<"RollNum : "<<venkat.rollNum<<endl;
    cout<<"SemPercentage : "<<venkat.semPerentage<<endl;
    cout<<"CollegeName : "<<venkat.collegeName<<endl;
    cout<<"CollegeCode : "<<venkat.collegeCode<<endl;
    return 0;
}

File: week4/Student.java
import java.util.Scanner;

class Student019
{
  public
    String fullName;
    int rollNum;
    double semPercentage;
    String collegeName;
    int collegeCode;
    Student019()
    {
      System.out.println("Student class is created successfully!!\nIam a student\n");
    }
    protected void finalize() throws Throwable
    {
      System.out.println("The class is finalized\n");
    }
}

public class Student
{
  public static void main(String[] args)
  {
    Student019 venkat=new Student019();;
    Scanner input=new Scanner(System.in);
    System.out.print("Enter your sweet name : ");
    venkat.fullName=input.nextLine();
    System.out.print("Enter your rollNumber : ");
    venkat.rollNum=input.nextInt();
    System.out.print("Enter your semPercentage : ");
    venkat.semPercentage=input.nextDouble();
    input.nextLine();
    System.out.print("Enter your collegeName : ");
    venkat.collegeName=input.nextLine();
    System.out.print("Enter your collegeCode : ");
    venkat.collegeCode=input.nextInt();
    System.out.print("Name : "+venkat.fullName+"\n");
    System.out.print("RollNumber : "+venkat.rollNum+"\n");
    System.out.print("SemPercentage : "+venkat.semPercentage+"\n");
    System.out.print("CollegeName : "+venkat.collegeName+"\n");
    System.out.print("CollegeCode : "+venkat.collegeCode+"\n");
    input.close();
  }
}
